sway-lib-std/src/storage.sw

```diff
@@ -4,8 +4,7 @@ use ::hash::sha256;
 use ::context::registers::stack_ptr;
 
 /// Store a stack variable in storage.
-#[storage(write)]
-pub fn store<T>(key: b256, value: T) {
+#[storage(write)]pub fn store<T>(key: b256, value: T) {
```

Comment:
Do we have to live with these attributes formatted to be on the same line for now?

Reply:
Fmt v2 fixes this!